README.md

```diff
@@ -63,7 +63,7 @@ If you've installed with rustup, make sure Rust is up to date by running
 
 User settings are currently stored in files; the general preferences are
 located at `~/Library/Application Support/XiEditor/preferences.xiconfig`.
-This file can be opened from File > Preferences (⌘ + ,).
+This file can be opened from `File` > `Preferences` (⌘ + ,).
```

Comment:
While you're at it I'd also use the button style for the shortcut keys.

Reply:
Specifically, the `<kbd>` html tag:
```
<kbd>CTRL</kbd>+<kbd>Z</kbd>
```
which renders like this:
<kbd>CTRL</kbd>+<kbd>Z</kbd>